### PR TITLE
Intial HDFS universe configs

### PR DESCRIPTION
**Pull request by adam-mesos**
Ready for initial sanity review (yet to be tested). @ConnorDoyle PTAL

### CODE REVIEW

File: repo/packages/H/hdfs/0/config.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "type": "string"
+    },
+    "hdfs/zkfc.ha.zookeeper.quorum": {
+      "description": "The zookeeper URL(s) to fill into hdfs-site.xml for the NameNodes and DataNodes.",
+      "type": "string"
+    },
+    "hdfs/scheduler-uri": {
+      "description": "The URI of the hdfs-mesos tarball, which will be extracted and used to run the scheduler.",
+      "type": "string",
+      "default": "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/ovh/hdfs-mesos-0.1.0.tgz"
```
Comment (adam-mesos) [inline repo/packages/H/hdfs/0/config.json]: This is the binary with config bundled in for our OVH cluster, but we'll probably want a more generic default (and easy overrides)

**Comment by adam-mesos** [comment]
@ConnorDoyle Updated now that it actually works for me. PTAL

**Comment by ConnorDoyle** [comment]
Will merge this as soon as #37 lands here.

**Comment by ConnorDoyle** [comment]
:guitar: :rocket: :confetti_ball: :beers:

**Pull request merged**